Gemfile

```diff
@@ -37,20 +36,21 @@ gem 'neo4j', '~> 9.6.2'
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'byebug', platforms: %i[mri mingw x64_mingw]
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '~> 3.39'
-  gem 'selenium-webdriver'
   gem 'neo4j-rake_tasks'
+  gem 'selenium-webdriver'
+  gem 'sorted_set'
```

Comment:
Was told to install the sorted_set gem in a previous [CI run](https://github.com/rastamhadi/tomodachi/actions/runs/4925592704/jobs/8800007671#step:7:121).

```
** Execute neo4j:migrate:all
rails aborted!
The `SortedSet` class has been extracted from the `set` library.
You must use the `sorted_set` gem or other alternatives.
```